src/System.Net.Primitives/tests/FunctionalTests/SocketAddressTest.cs

```diff
@@ -76,13 +76,21 @@ public static void ToString_Expected_Success(AddressFamily family, int size, boo
         }
 
         [Theory]
-        [InlineData((AddressFamily)12345, -1)]
-        [InlineData((AddressFamily)12345, 16)]
+        [InlineData((AddressFamily)123456, -1)]
```

Comment:
These number (123456) throw because they are greater than 65538?  Why not just use 65539 or something? It's hard for someone new to this codebase to understand why these particular numbers were chosen for the test.

Reply:
updated.